.github/scripts/compare-benchmarks.py

```
@@ -0,0 +1,429 @@
+            regressions.append(
+                (name, baseline_result, current_result, change_pct, severity)
+            )
+            if severity_rank[severity] > severity_rank[max_severity]:
+                max_severity = severity
+        elif severity == "IMPROVEMENT":
+            improvements.append((name, baseline_result, current_result, change_pct))
+
+    # Generate markdown
+    md = f"""# Performance Review Results
+
+**Date**: {datetime.now().strftime('%Y-%m-%d %H:%M:%S UTC')}
```

Ruff (0.14.3)

`datetime.datetime.now()` called without a `tz` argument

(DTZ005)